Packs/Okta/Playbooks/playbook-Okta_-_User_Investigation.yml

```diff
@@ -0,0 +1,1363 @@
+          "y": -880
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: bb352d99-3d2e-4ae2-8b9b-4ecb8e2a970c
+    type: title
+    task:
+      id: bb352d99-3d2e-4ae2-8b9b-4ecb8e2a970c
+      version: -1
+      name: Permanent Location  Check
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1620,
+          "y": 820
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 08b0a87e-8ab1-4235-8797-1e656dd0e9c5
+    type: condition
+    task:
+      id: 08b0a87e-8ab1-4235-8797-1e656dd0e9c5
+      version: -1
+      name: Is Okta V2 integration enabled?
+      description: Checks if Okta V2 integration enabled
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "18"
+      "yes":
+      - "9"
+      - "33"
+      - "38"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Okta v2
+                    ignorecase: true
+                accessor: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 600,
+          "y": 75
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: df7a7cad-4048-4344-8c25-e59d5ce8d039
+    type: condition
+    task:
+      id: df7a7cad-4048-4344-8c25-e59d5ce8d039
+      version: -1
+      name: The user work from a permanent country?
+      description: Check if the user work from a permanent country
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "6"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: in
+          left:
+            value:
+              complex:
+                root: inputs.LoginCountry
```

Comment:
What will happen if the LoginCountry input is empty? you will add the output False even tho the check cant happen.

Reply:
Done.